test/unit/v0/test_api.py

```diff
@@ -47,6 +48,18 @@ def test_list_buckets(self):
                         'mode': 'none'
                     },
                 },
+            'fileLockConfiguration':
+                {
+                    'isClientAuthorizedToRead': True,
+                    'value':
```

Comment:
is this something that server would return?

Reply:
yes